code/game/gamemodes/game_mode.dm

```diff
@@ -97,16 +99,16 @@
 ///convert_roundtype()
 ///Allows rounds to basically be "rerolled" should the initial premise fall through
 /datum/game_mode/proc/convert_roundtype()
-	var/living_crew = 0
+	var/list/living_crew = list()

```

Comment:
Later in this proc living crew was being used for this check:

for(var/mob/living/carbon/human/H in living_crew)

Which is functionally equivalent to saying "for all humans in 8".

What I want to point out here, and what terrifies me slightly, is that somehow, in that state, this <b>WORKED</b>.